src/Narochno.Primitives/Parsing/Convert.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+
+namespace Narochno.Primitives.Parsing
+{
+    public static class Convert<TType>
+        where TType : struct
+    {
+
+        private static IParser<TType> parser;
+
+        static Convert()
+        {
+            Refresh();
+        }
+
+        /// <summary>
+        /// Refreshes the parser from the DefaultParserLibrary if it is changed.
+        /// </summary>
+        /// <exception cref="ArgumentNullException"></exception>
+        public static void Refresh()
+        {
+            parser = DefaultParserLibrary.Instance.GetParser(typeof(TType)) as IParser<TType>;
```

Comment:
I'm trying to avoid constant reflection.  I guess it's not a huge deal.

I'll just not cache.